src/layouts/default.vue

```diff
@@ -98,27 +98,18 @@ export default {
       description: state => state.description,
       action:      state => state.action
     }),
-    defaultRoutes() {
+    platformSpecificRoute() {
+      return os.platform().startsWith('win') ? '/Integrations' : '/ApplicationSettings';
```

Comment:
I didn't realize `ApplicationSettings` only happened with lima; that might need a better name… but not in this PR.

Ideally we'd call it `/WSLIntegrations` instead, though, to be clearer that it's Windows only.